dataflux_core/tests/fake_gcs.py

```diff
@@ -57,8 +57,12 @@ def list_blobs(
         return results
 
     def blob(self, name: str):
+        missing_path = False
```

Comment:
Probably want to name this the same as the keyword arg its provided to for clarity (including the string passed in)